kibbeh/src/global-stores/useScrollStore.ts

```diff
@@ -0,0 +1,16 @@
+import create from "zustand";
+import { combine } from "zustand/middleware";
+
+export const useScrollStore = create(
```

Comment:
what is this about? Remove console.logs related to this (you should be splitting the prs so I can merge the ones that don't need changes)

Reply:
this was to fix #2212, but it doesn't work as i expected... ill just remove the code for it then